src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -18,8 +18,8 @@
   
 
   // Main
-  --#{$text-input-group}__main--first-child--not--text-input--MarginInlineStart: var(--pf-t--global--spacer--xs);
-  --#{$text-input-group}__main--m-icon__text-input--PaddingInlineStart: var(--pf-t--global--spacer--xl);
+  --#{$text-input-group}__main--first-child--not--text-input--MarginInlineStart: calc(var(--pf-t--global--spacer--control--horizontal--plain) / 2);
```

Comment:
`padding-block-start: var(--#{$text-input-group}__main--first-child--not--text-input--PaddingBlockStart);` and `padding-block-end: var(--#{$text-input-group}__main--first-child--not--text-input--PaddingBlockEnd);` on line 158-9 are undefined - should it they defined here?

Reply:
Originally those padding-inlines were margins for the label group, but since we have the block on line ~163 for the padding on label group, removing the padding properties on lines 158-9.